Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest
```

Comment:
Let's lock this `alpine:3.15`. I prefer to be more intentional about base image changes.